src/PythonMigrationViewExtension/Properties/Resources.resx

```diff
@@ -170,6 +170,12 @@ By clicking “Continue”, you are agreeing to implement the syntax changes to
   <data name="MigrationAssistantNoChangesStateMessage" xml:space="preserve">
     <value>Your code is ready to run with the Python 3 engine. Click accept and we'll switch to the Python 3 engine for you.</value>
   </data>
+  <data name="MigrationAssistantErrorStateHeader" xml:space="preserve">
+    <value>There is an error in your code!</value>
+  </data>
+  <data name="MigrationAssistantErrorStateMessage" xml:space="preserve">
+    <value>Fix all errors before attempting code migration. Clicking cancel will switch back to the Python 2 engine without any code changes.</value>
```

Comment:
Technically, the engine will not be switched back. It should simply remain the same. Even though it doesn't make much sense, you could also launch the migrator with the `CPython3` engine set.

How about something more on the lines of: `The code migration cannot continue. Please click cancel and fix all errors before re-attempting it.`

Reply:
I'll file a new task - but we should disable the `2->3` button when you have the CPython3 engine set. It's confusing otherwise!

Reply:
If the engine is already CPython, I'm not doing anything special currently like switching it back to IronPython. So I'm removing the second sentence altogether and leaving it at that.